tests/test_ftlid.py

```diff
@@ -36,3 +39,38 @@ def test_prediction_multiple_langs_with_prob():
     assert lang[1] == 'de'
     assert len(p) == 2
     assert p[0] > 0.5
+
+
+def test_model_pre_loaded():
```

Comment:
Thanks a ton for these tests @swecooo!

I realize what sort of an enormous advantage does mocking provide but could you please also create a test or two where you'd test this out end-to-end with real model files? I realize that the main objective of mocking is making sure the tests are fast but in our case that should not be such a big deal.

if you think otherwise, please feel free to ignore this and just let me know :)

Reply:
Well, I think mocked tests are also easier when you need to test various exceptional cases (that might be hard to simulate in end-to-end tests). However in this case we can simulate them easily so I will change it to end-to-end fashion where possible. :slightly_smiling_face: 

- `test_model_pre_loaded`: This test actually doesn't change any behavior, it just asserts that the method was not called.
- `test_provide_model`: I will change this one to load a real model.

Reply:
@swecooo No need to change anything, I would just add a few more and be done with it.

Reply:
I see, I'm sorry, I didn't read carefully enough. I added 2 end-to-end tests which should cover some cases. :slightly_smiling_face: